tests/test_download_cli_integration.py

```
@@ -27,16 +27,42 @@ def test_cli_integration_main_loads_config_and_runs(mocker):
     )
 
     result = integration.main(config=config)
 
     run_download.assert_called_once_with({"DOWNLOAD_DIR": "/tmp"}, False)
-    assert result[0] == ["fw"]
-    assert result[5] == "fw_latest"
+
+    # Unpack the 13-field tuple into named locals
+    (
+        downloaded_firmwares,
+        new_firmware_versions,
```

Ruff (0.15.7)

[warning] Unpacked variable `new_firmware_versions` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)